src/Action.hs

```diff
@@ -86,6 +89,21 @@ module Action
 
     , module Action.Smtp
     , sendMailToRole
+
+    -- * moderator's event log
```

Comment:
Probably.

In any case, *some* re-structure of `Action` wlll be due soon.  And of `Pure`.